commodore/catalog.py

```diff
@@ -84,14 +102,20 @@ def update_catalog(cfg, target_name, repo):
         click.echo(textwrap.indent(commit_message, '   '))
     if changed:
         if not cfg.local:
+            if cfg.interactive:
+                if yes_or_no('Should the push be done?'):
+                    cfg.push = True
+                else:
+                    cfg.push = False
+
             if cfg.push:
```

Comment:
Let's combine the flag with `--push`, so that both `--push --interactive` flags need to be specified.

Reply:
so, confirmation should be asked for both --push and --interactive?

Reply:
no, but --interactive only has an effect together with --push (i.e. only --interactive does nothing)